sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -155,18 +155,13 @@ private static async ValueTask<(AccessToken, TokenCredential)> GetTokenFromSourc
 
                     return (token, sources[i]);
                 }
-                catch (AuthenticationFailedException e)
+                catch (CredentialUnavailableException e)
                 {
                     exceptions.Add(e);
                 }
-                catch (Exception e) when (!(e is OperationCanceledException))
-                {
-                    exceptions.Add(e);
-                    throw AuthenticationFailedException.CreateAggregateException(UnhandledExceptionMessage + e.Message, exceptions);
-                }
```

Comment:
Exceptions here still need to be wrapped we just don't want to aggregate. I think this catch is still necessary but it  should just wrap and  throw for all exceptions which are not `OperationCancelledException`

Reply:
It is wrapped by `GetTokenImplAsync`